tests/Toolkit/ATest.php

```diff
@@ -447,6 +447,26 @@ public function testFill()
 			'elephant',
 			'elephant'
 		], A::fill($array, 5, 'elephant'));
+
+		// Callable
+		$this->assertSame([
+			'miao',
+			'wuff',
+			'tweet',
+			'elephant',
+			'elephant',
+			'elephant'
+		], A::fill($array, 6, function () {
+			return 'elephant';
+		}));
```

Comment:
Let's use arrow functions here, e.g.
```php
A::fill($array, 6, fn () => return 'elephant');
```


But also below.

Reply:
Yeah, i am a bit misty about what php version introduced some modern features i use in my code, so i didn’t want to be overzealous in far reaching kirby code, even though this is a test. Will update